harness/determined/experimental/client.py

```diff
@@ -66,6 +66,8 @@
     TrialOrderBy,
     TrialReference,
     TrialSortBy,
+    CheckpointSortBy,
```

Comment:
**question**: why are we importing these here when they're not used in the remainder of the file?

Reply:
since it's a type the user is expected to pass in, we want the user to be able to import them.